### DISCUSSION TITLE
XHTML (javascript version)

### DISCUSSION THREAD

**Discussion by JonPolish** [freeplane/freeplane#910] [Help]
Exporting maps to this format is almost perfect for me. But there is one thing I cannot figure out. Mousing over a node will display its associated note, but there is no way to keep the window open (unless I am missing something). Can this be improved? There is no way to click any links contained within the note.

Thank you.

Jon

EDIT:
Additional information. This was tested on MS Edge, Chrome, Firefox, and IE. The results are the same as detailed above. Are others seeing this?

EDT #2
Additional information: Freeplane 1.10.6 under Windows 10.

**Comment by euu2021** [comment]
I see. The only way that I managed to select the text is by using Ctrl+A (select all).

One idea is to create an AutoHotkey script that does something like this:

- user selects all text and copy
- script identifies that it is from a map
- script reads the clipboard and identifies the text inside the note
- script does something with that link (eg, open on a new tab in the browser)

**Reply by euu2021** [reply]
I found a trick!

![chrome_m13w3YGOsL](https://user-images.githubusercontent.com/77707706/216429020-7145f0c7-798a-4150-b876-10a7084c6082.gif)

Select the text of the node and then drag it. The note won't disappear! 🤣

**Reply by JonPolish** [reply]
Thank you.

This is good to know, but when using this with colleagues, it is a problematic work around. Better would be to fix it so it works as expected. It is so close to what I need...

Jon
